_posts/2017-10-18-Ordina-Architecture-Kata.md

```diff
@@ -160,8 +161,14 @@ Our kata for today — AM.I.SCK
  - Enable parts of the system for direct patient usage
  - Conversations are not considered medical records
 
- The sixteen attendees were divided in groups of four.
- We had to draw the different architectural views for the AM.I.SCK platform.
+The sixteen attendees were divided in groups of four.
+We had to draw the different architectural views for the AM.I.SCK platform.
```

Comment:
We had to create ...

I emphasised the drawing too much during the theory. It's not only about diagraming.

Reply:
Sentence removed, I said the same in the next paragraph.